applications/simulate_prod.py

```diff
@@ -0,0 +1,332 @@
+            "repetition."
+        ),
+        type=int,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--run",
+        help="Run number (actual run number will be 'start_run' + 'run')",
+        type=int,
+        required=True,
+    )
+    config.parser.add_argument(
+        "--data_directory",
+        help="The directory where to save the corsika-data and simtel-data output directories.",
+        type=str.lower,
+        required=False,
+        default="./",
+    )
+    return config.initialize(db_config=True, telescope_model=True)
+
+
+def _proccess_simulation_config_file(config_file, primary_config, logger):
+    """
+    Read the simulation configuration file with all the details
+    on shower and array simulations.
+
+    Attributes
+    ----------
+    config_file: str
+        Name of simulation configuration file.
+    primary_config: str
+        Name of the primary selected from the configuration file.
+
+    Returns
+    -------
+    str
+        Label of simulation configuration.
+    dict
+        Configuration of shower simulations.
+    dict
+        Configuration of array simulations.
+
+    """
+
+    try:
+        with open(config_file) as file:
+            config_data = yaml.load(file)
+    except FileNotFoundError:
+        logger.error(f"Error loading simulation configuration file from {config_file}")
+        raise
+
+    label = config_data.pop("label", "test_run")
+    default_data = config_data.pop("default", {})
+
+    for primary, primary_data in config_data.items():
+
+        if primary_config is not None and primary != primary_config:
+            continue
+
+        this_default = copy(default_data)
+
+        config_showers = copy(this_default.pop("showers", {}))
+        config_arrays = copy(this_default.pop("array", {}))
+
+        # Grabbing common entries for showers and array
+        for key, value in primary_data.items():
+            if key in ["showers", "array"]:
+                continue
+            config_showers[key] = value
+            config_arrays[key] = value
+
+        # Grabbing showers entries
+        for key, value in primary_data.get("showers", {}).items():
+            config_showers[key] = value
+        config_showers["primary"] = primary
+
+        # Grabbing array entries
+        for key, value in primary_data.get("array", {}).items():
+            config_arrays[key] = value
+        config_arrays["primary"] = primary
+
+        # Filling in the remaining default keys
+        for key, value in this_default.items():
+            config_showers[key] = value
+            config_arrays[key] = value
+
+    config_arrays["data_directory"] = config_showers["data_directory"]
+    config_arrays["site"] = config_showers["site"]
+    config_arrays["layout_name"] = config_showers["layout_name"]
+
+    return label, config_showers, config_arrays
+
+
+def _translate_from_azimuth_direction_to_phi(logger, from_azimuth_direction):
```

Comment:
since you have to translate it anyways, I would also support the option of passing directly the azimuth angle. I know that we dont need that for e.g. prod6 but it is strange to me not to allow it

Reply:
It will be done, but not now (see statement above).